ntbs-service-unit-tests/DataMigration/CaseManagerImportServiceTests.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using ntbs_service.DataAccess;
+using ntbs_service.DataMigration;
+using ntbs_service.DataMigration.RawModels;
+using ntbs_service.Models.Entities;
+using ntbs_service.Models.ReferenceEntities;
+using Xunit;
+
+namespace ntbs_service_unit_tests.DataMigration
+{
+    public class CaseManagerImportServiceTests : IClassFixture<CaseManagerImportServiceTests.DatabaseFixture>
```

Comment:
I think we're missing a few test cases here. I think we need tests for:
- User not imported if already active in NTBS
- TB Service not added when legacy user does not have permissions

There may be others that you can think of